desktop-widgets/divelistview.cpp

```diff
@@ -811,17 +813,17 @@ void DiveListView::contextMenuEvent(QContextMenuEvent *event)
 		}
 	}
 	if (d) {
-		popup.addAction(tr("Delete dive(s)"), this, &DiveListView::deleteDive);
+		popup.addAction(tr("Delete %n dive(s)","",amount_selected), this, &DiveListView::deleteDive);
```

Comment:
I'm curious how you decided where to add the `%n` and where not to add it.
This obviously could have been mentioned in the menu entries that add dive(s) to a trip, right?
Not saying that this is wrong, just trying to understand why here not not in the other cases.

Reply:
Good pickup.
Pretty sure that was just missed as I was getting my head around the translation system. The %n parameter isn't required in the source string as passed into tr(), as its added in the translation lookup. TBH, it seems strange to me that adding the %n to the source string even worked at all - or maybe my assumptions about how the translation magic works are wrong.

I've added another commit to bring the "Delete dive(s)" back in line with the rest.

I'll add here that my intent is to include the `amount_selected` value as a parameter, but not actually use the number itself in the translations (just use it to determine the singular/plural). 

It feels a bit weird from a UI perspective to fill the context menu with "Delete 4 dives", "Add 4 dives to trip", etc.
Regardless, the choice of whether to include the number or not can lie within the translation, not within the code.

Reply:
You are correct, a translator who KNOWS that this is the case could add the %n - but I'd find that weird, TBH, and of course it isn't discoverable to the translator unless they are very familiar with the code as well.